charts/eks/values.yaml

```diff
@@ -121,8 +121,30 @@ syncer:
   replicas: 1
   # NodeSelector used to schedule the syncer
   nodeSelector: {}
-  # Affinity to apply to the syncer deployment
-  affinity: {}
+  # Affinity to apply to the vcluster statefulset
+  affinity:
+    podAntiAffinity:
+      preferredDuringSchedulingIgnoredDuringExecution:
+      # if possible avoid scheduling more than one pod on one node
+      - weight: 100
+        podAffinityTerm:
+          labelSelector:
+            matchExpressions:
+            - key: app
+              operator: In
+              values:
+              - vcluster
+          topologyKey: "kubernetes.io/hostname"
+      # if possible avoid scheduling pod onto node that is in the same zone as one or more vcluster pods are running
+      - weight: 50
+        podAffinityTerm:
+          labelSelector:
+            matchExpressions:
+            - key: app
+              operator: In
+              values:
+              - vcluster
```

Comment:
Same issue as with label selector in PDB.
For the labelSelector in both affinity rules above.